libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Actions/Microsoft.SwitchCondition.schema

```diff
@@ -11,30 +11,34 @@
         "condition": {
```

Comment:
>condition [](start = 9, length = 9)

Hmmm..this name looks wrong. I wonder if we should be calling it switch(PROPERTY){ case VALUE,
This would align it with SetProperty() and GetProperty() and naming conventions we have used on various pieces.

Reply:
Yes. I like that. Filed a DCR to track that. For now, I'll update the description to refer to 'property' instead of 'condition'